libs/wingc/src/type_check/jsii_importer.rs

```diff
@@ -794,20 +776,22 @@ impl<'a> JsiiImporter<'a> {
 				self.setup_namespaces_for(&type_fqn);
 			}
 		} else {
-			// No submodules, so lets manually setup a root namespace for the module
-			let ns = self.wing_types.add_namespace(Namespace {
-				name: assembly.name.clone(),
-				env: SymbolEnv::new(None, self.wing_types.void(), false, Phase::Preflight, 0),
-			});
-			self
-				.wing_types
-				.libraries
-				.define(
-					&Symbol::global(assembly.name.clone()),
-					SymbolKind::Namespace(ns),
-					StatementIdx::Top,
-				)
-				.expect("Failed to define jsii root namespace");
+			// println!("defining root namespace for {}", assembly.name);
```

Comment:
Seems like this is just creating trouble when I am trying to import `constructs` as a different alias.